pygromos/files/blocks/imd_blocks.py

```diff
@@ -254,7 +394,7 @@ class NEW_REPLICA_EDS(_generic_imd_block):
     CONT: bool
     PERIODIC: int
 
-    _order = [[["REEDS"], ["NRES", "NUMSTATES", "NEOFF"], ["RES(1 ... NRES)"],
+    _order = [[["REEDS"], ["NRES", "NEOFF", "NUMSTATES"], ["RES(1 ... NRES)"],
```

Comment:
I'm not sure about the order here! I think I should check the source code of gromosXX for this @SchroederB

Reply:
I am pretty convinced of that  ["NRES", "NEOFF", "NUMSTATES"] is the correct order, but please doublecheck. It "just" affects the labels in the *imd, but still...